pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -14,7 +15,7 @@ let
       initConfig,
       updateUrl,
       license ? lib.licenses.mit,
-      maintainers ? [ lib.maintainers.phanirithvij ],
+      maintainers ? dprint.meta.maintainers,
```

Comment:
Personally, I’m not using dprint plugins from nixpkgs at the moment. Would it be possible to make the default empty? :bow:

Reply:
reverted